packages/search/tests/unit/services/navi-search/navi-definition-search-provider-test.js

```diff
@@ -0,0 +1,119 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import Pretender from 'pretender';
+import metadataRoutes from '../../../helpers/metadata-routes';
+
+let Service, Server;
+
+module('Unit | Service | navi-definition-search-provider', function(hooks) {
+  setupTest(hooks);
+  setupMirage(hooks);
+
+  hooks.beforeEach(async function() {
+    await this.owner.lookup('service:bard-metadata').loadMetadata();
+    Server = new Pretender(metadataRoutes);
+    Service = this.owner.lookup('service:navi-search/navi-definition-search-provider');
+  });
+
+  hooks.afterEach(async function() {
+    Server.shutdown();
+  });
+
+  test('search definition of a dimension', async function(assert) {
+    assert.expect(3);
+    metadataRoutes.bind(Server);
+    const results = await Service.search.perform('age');
+
+    const expectedResults = [
+      {
+        id: 'age',
+        name: 'Age'
+      }
+    ];
+
+    assert.equal(results.component, 'navi-search-result/definition', 'Result contains correct display component name');
+    assert.equal(results.title, 'Definition', 'Result contains correct title for the search result section');
+    results.data.forEach((result, index) => {
+      assert.ok(
+        Object.keys(expectedResults[index]).every(key => result[key] === expectedResults[index][key]),
+        'Result contains the the expected properties and values'
+      );
+    });
```

Comment:
```suggestion
    assert.deepEqual(result.data.map(res => ({ id: res.id, name: res.name})), expectedResults, 'Result contains the the expected properties and values');
```
How about this? It's not a blocker, but I feel looping through an array and having a separate assert statement for each index is unnecessary in this case.

Reply:
ohh yes it's better this way